### DISCUSSION TITLE
Получить статус закрепления (пина) для ячейки

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#281] [Как это сделать?] [answered]
Есть ли возможность обратиться к полю, получить, что оно запиненное и, соответственно, не вычислять кодом значение в этой ячейке? Может какие-то альтернативные способы?

Для оптимизации больших расчетных таблиц такое думал использовать, чтоб не пересчитывать часть полей при этом сохранить текущие значения в поле.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
#### Получение значений зафиксированных полей

Возможно получить признак фиксирования значения для полей с кодами — `true` или `false` используя `#h.fieldname`.

Для зафиксированных полей можно получить расчетное значение через `#c.fieldname`.

Тут такой момент, что в коде нельзя получить информацию о запиненности тк код еще не рассчитан.

Можно сетить при наличии пина техническое поле ранее и тогда:

```
=: if(condition: $recalc_fix = true; then: $#onfv; else: $calc)

calc: ...

```
